csrc/optimization/alias_analysis.cpp

```diff
@@ -132,6 +132,8 @@ void AliasFinder::handle(const ViewOp* view) {
 
   std::unordered_map<IterDomain*, IterDomain*> in_rfactor_to_out_root =
       PairwiseRootDomainMap(in, out).mapBroadcast(true).mapProducerToConsumer();
+  std::unordered_map<IterDomain*, IterDomain*> out_root_to_in_rfactor =
```

Comment:
nitpick: Please add a comment here explaining why we need to map everything to in_rfactor to detect the correct expand flag. A link to the issue would be great as well!

Reply:
The comment you are looking for was at Line 151.